lib/archethic/bootstrap/network_init.ex

```diff
@@ -213,7 +189,7 @@ defmodule Archethic.Bootstrap.NetworkInit do
 
   @spec self_replication(Transaction.t()) :: :ok
   def self_replication(tx = %Transaction{}) do
-    :ok = Replication.validate_and_store_transaction_chain(tx)
+    :ok = Replication.validate_and_store_transaction_chain(tx, true)
```

Comment:
I think to avoid conflicts with others PR, you can use `Replication.sync_transaction_chain` as the flag will be removed.